datajoint/schema.py

```diff
@@ -41,6 +85,20 @@ def __init__(self, database, context, connection=None):
                 raise DataJointError("Database named `{database}` was not defined, and"
                                      " an attempt to create has failed. Check"
                                      " permissions.".format(database=database))
+        else:
+            class_factory = SchemaFactory()
+            classes = []
```

Comment:
What is the function of the variable `classes` here?  It does not seem to be used.